lib/iex/test/iex/autocomplete_test.exs

```diff
@@ -285,4 +285,20 @@ defmodule IEx.AutocompleteTest do
     eval("struct = %IEx.AutocompleteTest.MyStruct{}")
     assert expand('struct.my') == {:yes, '_val', []}
   end
+
+  test "ignore invalid Elixir module literals" do
+    defmodule :"Elixir.José", do: nil
```

Comment:
Let's call it `IEx.AutocompleteTest.Unicodé`. Modules are global and there is a chance someone may define `José` elsewhere. :)